feat(UI-1004): redesign sidebar navigation

J1za opened the pull request:
## Description

1. More delicate and new icons
2. Different spacing

## Linear Ticket
https://linear.app/autokitteh/issue/UI-1004/main-navigation-menu-re-design
## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

<!--
     For a timely review/response, please avoid force-pushing additional
     commits if your PR already received reviews or comments.
     Before submitting a Pull Request, please ensure you've done the following:
     - 👷‍♀️ Create small PRs. In most cases this will be possible.
     - ✅ Provide tests for your changes.
     - 📝 Use descriptive commit messages (as described below).
     - 📗 Update any related documentation and include any relevant screenshots.
     Commit Message Structure (all lower-case):
     <type>(optional ticket number): <description>
     [optional body]
-->

RonenMars commented on src/assets/image/sidebar/index.ts:
```diff
@@ -1,5 +1,8 @@
+export { default as CircleQuestionIcon } from "@assets/image/sidebar/CircleQuestion.svg?react";
 export { default as ConnectionsIcon } from "@assets/image/sidebar/Connections.svg?react";
 export { default as DashboardIcon } from "@assets/image/sidebar/Dashboard.svg?react";
+export { default as FileIcon } from "@assets/image/sidebar/File.svg?react";
```
Please mention the source of each icon (link to the figma)

J1za replied:
added

RonenMars commented on src/assets/image/sidebar/index.ts:
```diff
@@ -1,5 +1,8 @@
+export { default as CircleQuestionIcon } from "@assets/image/sidebar/CircleQuestion.svg?react";
```
Move the **sidebar** folder to the icons folder.

J1za replied:
moved to the icons folder

RonenMars commented on src/components/organisms/sidebar/sidebar.tsx:
```diff
@@ -144,7 +144,7 @@ export const Sidebar = () => {
 
 						<Button className="hover:bg-green-200" href="/intro" title={t("intro")}>
-							<IconSvg className="size-7 transition" src={HelpIcon} />
+							<IconSvg className="size-5.5 ml-0.5 transition" src={CircleQuestionIcon} />
```
You don't have 5.5 size in `tailwind.config.cjs`

J1za replied:
added

RonenMars reviewed (commented):
A couple of comments and we're good to go.

RonenMars commented on src/components/organisms/sidebar/sidebar.tsx:
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable tailwindcss/no-custom-classname */
```
When you fix the `size-5.5` you won't need this line

J1za replied:
fixed

J1za commented:
> Code:
> 
> <img alt="image" width="60" src="https://private-user-images.githubusercontent.com/7413072/394868374-18d85a86-9ae8-43d8-957d-ab2740ae79ae.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwMjk3NTcsIm5iZiI6MTczNDAyOTQ1NywicGF0aCI6Ii83NDEzMDcyLzM5NDg2ODM3NC0xOGQ4NWE4Ni05YWU4LTQzZDgtOTU3ZC1hYjI3NDBhZTc5YWUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTJUMTg1MDU3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTlkMmY4OWY2MmY4MWFhYTRhYzlmNTI0MmYwNjAyOTgzNzU5NDNlOGU4NmNkN2ZiZDQ2ZjlhNTMxYmNlNGFjNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.3d95o-dc3bZNsZuxzOAydXATvgOolYoBf4uS3odo3xw">
> Design:
> 
> ![image](https://private-user-images.githubusercontent.com/7413072/394868489-aaf30ed6-a62e-4b90-89d6-b94eef4e58de.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwMjk3NTcsIm5iZiI6MTczNDAyOTQ1NywicGF0aCI6Ii83NDEzMDcyLzM5NDg2ODQ4OS1hYWYzMGVkNi1hNjJlLTRiOTAtODlkNi1iOTRlZWY0ZTU4ZGUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTJUMTg1MDU3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MGZiMzE1NjkwMTRlOGQ4NDA5OWI5ZDM1MzczODI0MzcwMzdjMGQ1ZmQyYjcwMWYyNGQ5MjNjM2MyM2Q3NDAwMyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.ob_zeGwfmgKmjfC_A6F5MBWwamn8e4ytr7MJLOjNp4U)
> 
> Differences:
> 
> 1. The gap between the icons should be much smaller.
> 2. The space from the bottom in the code is 40px, and in the design is 47px.
> 3. On icon hover is not round, it's elliptical and should be round circle:
> 
> <img alt="image" width="189" src="https://private-user-images.githubusercontent.com/7413072/394869194-52fd01f5-7279-4ea2-8798-ceefe0e17a32.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwMjk3NTcsIm5iZiI6MTczNDAyOTQ1NywicGF0aCI6Ii83NDEzMDcyLzM5NDg2OTE5NC01MmZkMDFmNS03Mjc5LTRlYTItODc5OC1jZWVmZTBlMTdhMzIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTJUMTg1MDU3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDc3MDAxZTg2NDQ3MDM3NmZhYjczZDVhM2MyYWE3ZmE2MjIxMjM4ZmYyY2I4OWMxNDViYzZjMzE1NTA0ZjYwZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.81Xvkunzk_i6YchzLbojDGtGNVdJMRdjb7nR9ul-wOA">
> 5. Correct me if I'm wrong, but I think that the plus is not in the middle of the hover circle:
> 
> <img alt="image" width="190" src="https://private-user-images.githubusercontent.com/7413072/394869423-69de0da3-2633-46cd-94cb-d9146747cb1d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwMjk3NTcsIm5iZiI6MTczNDAyOTQ1NywicGF0aCI6Ii83NDEzMDcyLzM5NDg2OTQyMy02OWRlMGRhMy0yNjMzLTQ2Y2QtOTRjYi1kOTE0Njc0N2NiMWQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTJUMTg1MDU3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjhhZTUzMTFmMmQ0MmVjZDc5NjMzMTIyMjkyODcwMmFhMTUyODI1YjUxZjY0NWY2MGU4NjJkYTViMmVjMDQwZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.mBnuR83z00Zx4x2kVc9e3vqrXW-PmwvjJ9itn6xb1qI">
> 6. Space between the logo and the hamburger should be 32px, in the code it's 42px
>    Code
> 
> <img alt="image" width="62" src="https://private-user-images.githubusercontent.com/7413072/394870092-4ce7a007-c437-4135-8560-fda1ed94b7c9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwMjk3NTcsIm5iZiI6MTczNDAyOTQ1NywicGF0aCI6Ii83NDEzMDcyLzM5NDg3MDA5Mi00Y2U3YTAwNy1jNDM3LTQxMzUtODU2MC1mZGExZWQ5NGI3YzkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTJUMTg1MDU3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NmVjODUwMWJhZDRhZTc1ZjAwMGM4N2NjOWQxMGE4ZDY3Yjg0YTFjNTczYmM3MWRhNWM5NzUxODgwYTAxYzQzNSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.EavjfSxLpqLUqiaEBt2pWXHVN7tvlQQGNUzZBvGPKo4">
> Design:
> 
> ![image](https://private-user-images.githubusercontent.com/7413072/394870497-a4d95192-0fb5-43a2-bf82-28bfd7666d76.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwMjk3NTcsIm5iZiI6MTczNDAyOTQ1NywicGF0aCI6Ii83NDEzMDcyLzM5NDg3MDQ5Ny1hNGQ5NTE5Mi0wZmI1LTQzYTItYmY4Mi0yOGJmZDc2NjZkNzYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTJUMTg1MDU3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTgzM2VmNGVlZWViNGMyYjRmMTI5ZmJhMzA5MWQ1MzkxMzRiNzQzOGY2ZDdkODNhYmU3ZWM0ZGI4OTUyMzA2ZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.WNWsJ0JSpbvmp_VYP8RI5b2v4DqmAd0SSyGs0Bu86xE)

1. changed gap between the icons 
2.  space from the bottom in the code 48px, I show you
3. fixed
4. fixed
5. fixed Space between the logo and the hamburger should be 38px 
![image](https://github.com/user-attachments/assets/8d9a67bd-a428-406c-a847-ce6466fa41f7)

J1za commented:
> 1. The icons are not aligned to the center inside the button:
> 
> ![image](https://private-user-images.githubusercontent.com/7413072/395463645-cb3a702b-a1e1-45a2-80c6-c70eae3c17e4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwNzY3MTUsIm5iZiI6MTczNDA3NjQxNSwicGF0aCI6Ii83NDEzMDcyLzM5NTQ2MzY0NS1jYjNhNzAyYi1hMWUxLTQ1YTItODBjNi1jNzBlYWUzYzE3ZTQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTNUMDc1MzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OWE2YjE3ODQ4NDVlOTkxODI5ZTU0YWU3NWM5NmNhZTgzYzk5ZjQzMGZhZDYzNDZhNTM5MzhmNTEzMTJmMTVhYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.9o5eXg39d5ZxMZMwuF8Urz7XnSioZrrChXeazbCgDGU)
> 
> ![image](https://private-user-images.githubusercontent.com/7413072/395463730-1f13d919-ff4c-4557-8b95-0b91e8066147.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwNzY3MTUsIm5iZiI6MTczNDA3NjQxNSwicGF0aCI6Ii83NDEzMDcyLzM5NTQ2MzczMC0xZjEzZDkxOS1mZjRjLTQ1NTctOGI5NS0wYjkxZTgwNjYxNDcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTNUMDc1MzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODFiNzA2YjFkYjZlY2JjODA5NzM3YTIyOGJmNmY0N2QzZTY3NjI5MTliZGEwNzE5MjBmOTUyMzlmZTU3NTM5YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.XDf2do5hgQ9acxEnlAlKVq6LjQCUBg0dmXq7Me4UtlU)
> 
> 2. I see that all three icons are the same size (20x20px), but somehow, the System Log looks smaller. Can we make it a little larger so it will look the same as the two others?
> 3. The avatar icon looks bigger than the others + The spacing between the avatar and the help icon is not the same as the others:
> 
> ![image](https://private-user-images.githubusercontent.com/7413072/395464771-544c2539-6fd1-4e7e-937d-b9f839674657.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwNzY3MTUsIm5iZiI6MTczNDA3NjQxNSwicGF0aCI6Ii83NDEzMDcyLzM5NTQ2NDc3MS01NDRjMjUzOS02ZmQxLTRlN2UtOTM3ZC1iOWY4Mzk2NzQ2NTcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTNUMDc1MzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MmI3MWVkYTBmYjMyYTRlMWVlZjBjM2NiZDE5OTMzODUwMDIwN2M3NTk4YWQ1OTAxZTI2NjZmNGRkOTljM2UzNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.j6_VAra8g7_x9cqau8uHyMoRqqqq4L4sU_VUkrOukUg)

fixed
